### ISSUE TITLE
Full calendar data

### ISSUE THREAD

**Issue by dnhass** [tomasy/date-time-adapter#18]
Can you add full calendar data so I can use month, day, year, plus time.

Thanks.

**Comment by tomasy** [comment]
Time already exist in the build in Clock adapter so I will not do that.
Do you have some use case so I can understand what you want to do?

**Comment by MirzaAreebBaig** [comment]
Hey,
Yes i would like that feature too month, day
For example if i want my any device to get ON @9PM Monday & then i want that same device to get ON @10PM Tuesday & so on 
it will be the best thing in rules

**Comment by tomasy** [comment]
I agree with you feature request and I think they shall be implemented in the iot-gateway. But it has been hard to get attention for that.
I have stopped to use iot-gateway and maintain date-time-adapter. I created date-time-adapter because it you must be possible to set rules depending om sunset and weekdays to make it usable. I thought it was a just quick fix, but the functionality was never implemented in the core. There are also some annoying bugs that was never fixed.
Instead I use Home Assistant https://www.home-assistant.io/

**Comment by wfahle** [comment]
This has been addressed 8n the latest 1.0.8
